### ISSUE TITLE
Do you have any on click events/functions

### ISSUE THREAD

**Issue by distractoo** [antocami/Fract-assigment#5]

**Comment by antocami** [comment]
Yes . This game is basically about the mouse events, the menu is 100% mouse friendly and the game-play control is just levitating the player by holding the mouse to increase the height and releasing the mouse to decrease the height.
